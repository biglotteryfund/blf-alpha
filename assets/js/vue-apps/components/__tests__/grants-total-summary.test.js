/**
 * @jest-environment jsdom
 */
/* eslint-env jest */
import { mount } from '@vue/test-utils';
import GrantsTotalSummary from '../grants-total-summary.vue';

describe('GrantsTotalSummary', () => {
    it('should render', () => {
        const wrapper = mount(GrantsTotalSummary, {
            // https://github.com/vuejs/vue-test-utils/issues/918
            context: {
                props: {
                    totalResults: 190000,
                    totalAwarded: 123456789
                }
            }
        });

        expect(wrapper.element).toMatchSnapshot();
    });
});
