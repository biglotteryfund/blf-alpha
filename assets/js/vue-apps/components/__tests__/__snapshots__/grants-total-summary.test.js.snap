// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GrantsTotalSummary should render 1`] = `
<span
  role="alert"
>
  
    190,000 projects found
    
        totalling Â£123,456,789
    
</span>
`;
